numpy/core/tests/test_umath.py

```diff
@@ -1759,7 +1759,15 @@ def test_reduce(self):
 
 class TestBitwiseUFuncs:
 
-    bitwise_types = [np.dtype(c) for c in '?' + 'bBhHiIlLqQ' + 'O']
+    _all_ints_bits = [
+        np.dtype(c).itemsize * 8 for c in np.typecodes["AllInteger"]]
```

Comment:
The linter complains about this line being one character too long which is fine.

Reply:
Fixed in 15dd4fa